helm/cluster-api-bootstrap-provider-kubeadm/templates/deploy.yaml

```diff
@@ -28,22 +28,7 @@ spec:
     spec:
       containers:
       - args:
-        - --secure-listen-address=0.0.0.0:8443
-        - --upstream=http://127.0.0.1:8080/
-        - --logtostderr=true
-        - --v=10
-        image: {{ .Values.Installation.V1.Registry.Domain }}/{{ .Values.rbacproxy.image.name }}:{{ .Values.rbacproxy.image.tag }}
-        imagePullPolicy: IfNotPresent
-        name: kube-rbac-proxy
-        ports:
-        - containerPort: 8443
-          name: https
-          protocol: TCP
-        resources: {}
-        terminationMessagePath: /dev/termination-log
-        terminationMessagePolicy: File
-      - args:
-        - --metrics-addr=127.0.0.1:8080
+        - --metrics-addr=0.0.0.0:8080
```

Comment:
Do we need a network policy to allow this traffic?

Reply:
scraping is working in my tests so I guess no, we don't

Reply:
noice 🚀